08-rails/exercises/erd-practice-activity.md

```diff
@@ -0,0 +1,21 @@
+# ERD Practice Activity
+
+## Scenario
+
+We are setting up an application to allow people to share their homes to tourists online like AirBnb.  So our initial user stories would be:
+
+- As a user I can add a home to be listed for rentals, along with a price, so I can rent the home out and make money.
```

Comment:
I think "home or property" would be a good descriptor here. "listed for rentals" is not quite right, I think we want "listed for rent" here and on the line below. I don't know if we need the last portion